system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -81,6 +100,18 @@
       (:concept-id (ingest/ingest-concept
                     (assoc collection :concept-id (generate-concept-id x "PROV1")))))))
 
+(defn- ingest-collection-in-umm-json-format
+  "Ingest a collection in UMM Json format and return a list of one concept-id.
+   This is used to test the FIND_AND_REMOVE on platforms/instruments. Since differenct
+   formats keep different parts of the platforms/instruments, can't combine them together
+   to test what's found and what's removed."
+  [attribs]
+  (let [collection (data-umm-c/collection-concept
+                     (data-umm-c/collection 1 attribs)
+                     :umm-json)]
+    [(:concept-id (ingest/ingest-concept
+                   (assoc collection :concept-id (generate-concept-id 1 "PROV1"))))])) 
+
```

Comment:
I ran into a lot of issues with the tests.  First of all, (data/umm-c/collection-concept  collection format) doesn't work when format is iso* and collection contains list of Platforms/Instruments because it complains about smap-keyword-str not working.  Since it's not bulk-update related, I just skipped that. Then these other formats, they will remove different fields for platform/instrument, so you can't combine them in one test.  So I decided to just do one umm-json format.